src/neural/shared/activation.h

```diff
@@ -21,19 +21,20 @@
 #include <cstddef>
 #include <vector>
 
-#include "proto/net.pb.h"
-
 namespace lczero {
+// The following list matches the one in net.proto. Ideally this would be done
+// by including proto/net.pb.h, but this is incompatible with nvcc.
 enum ActivationFunction {
-  NONE = pblczero::NetworkFormat::ActivationFunction::NONE,
-  RELU = pblczero::NetworkFormat::ActivationFunction::RELU,
-  TANH = pblczero::NetworkFormat::ActivationFunction::TANH,
-  SIGMOID = pblczero::NetworkFormat::ActivationFunction::SIGMOID,
-  SELU = pblczero::NetworkFormat::ActivationFunction::SELU,
-  MISH = pblczero::NetworkFormat::ActivationFunction::MISH,
-  SWISH = pblczero::NetworkFormat::ActivationFunction::SWISH,
-  RELU_2 = pblczero::NetworkFormat::ActivationFunction::RELU_2,
-  SOFTMAX = pblczero::NetworkFormat::ActivationFunction::SOFTMAX
+  DEFAULT = 0,
```

Comment:
Maybe prefix them all with ACTIVATION or make them `enum class` instead of `enum`?
Because names like `DEFAULT` and `NONE` are quite generic and exposing them right into the `lczero` namespace without any class is not that nice.

Reply:
Those were already merged with #1858 but we can obviously revise them as needed. The names were like that before, and this is only included by backend code.

Reply:
I made https://github.com/LeelaChessZero/lc0/pull/1860 to discuss this.